eZ/Bundle/EzPublishCoreBundle/Resources/config/locale.yml

```diff
@@ -56,7 +58,7 @@ parameters:
 
     ezpublish.locale.browser_map:
         au: ['eng-AU']
-        be: ['fre-BE']
+        be: ['fre-BE', 'dut-BE']
```

Comment:
Shouldn't this also include the ger-BE you added?

Reply:
Absolutely!